docs/contents/community/faq.md

```diff
@@ -12,11 +12,46 @@
 
     We appreciate input and advice concerning features that are currently covered by the service broker only, for example bind/unbind or cataloging and looking forward to learn from the community how they are using service broker so that we can take this into account.
 
+## Cluster API
+
+!!! question "Question"
+    Does ACK replace [cluster api](https://github.com/kubernetes-sigs/cluster-api)?
```

Comment:
Should this be made more explicit as "Does the planned ACK service controller for EKS replace Kubernetes Cluster API?"